src/tv2_afvd_showstyle/migrations/index.ts

```diff
@@ -233,6 +234,13 @@ export const showStyleMigrations: MigrationStepShowStyle[] = [
 	 */
 	removeSourceLayer('1.7.5', 'AFVD', 'studio0_graphicsIdent_persistent'),
 
+	/**
+	 * 1.7.6
+	 * - Rename columns in Graphics Setups table
+	 */
+	renameTableColumn('1.7.6', 'GraphicsSetups', 'OvlShowName', 'OvlShowName'),
```

Comment:
Why do these use the same value for "old" and "new", am i missing something? :-)

Reply:
My mistake, thanks!